packages/ui/feature-builder-header/src/lib/publish-button/publish-button.component.ts

```diff
@@ -22,10 +22,10 @@ export class PublishButtonComponent implements OnInit {
   constructor(private store: Store) {}
 
   ngOnInit(): void {
-    this.setCollectionStateListener();
+    this.setFlowStateListener();
   }
 
-  private setCollectionStateListener() {
+  private setFlowStateListener() {
```

Comment:
If it's only used there, why not move it inside ngOnInit and remove this function?

Reply:
very long function and you will have to read it to understand what it is doing.